Makefile.am

```
@@ -258,16 +257,9 @@ CPP_LINT_FILES = $(shell find . \( -name "*.h" -or -name "*.cpp" \) -and ! \( \
         -wholename "./tools/ola_trigger/config.tab.*" -or \
         -wholename "./tools/ola_trigger/lex.yy.cpp" \) | xargs)
 .PHONY : cpplint
-if FOUND_CPPLINT
 cpplint: Makefile.am
-	cpplint.py --filter=$(CPP_LINT_FILTER) $(CPP_LINT_FILES)
+if FOUND_CPPLINT
+	cpplint --repository=. --filter=$(CPP_LINT_FILTER) $(CPP_LINT_FILES)
 else
-cpplint: Makefile.am cpplint.py
-	./cpplint.py --filter=$(CPP_LINT_FILTER) $(CPP_LINT_FILES)
-endif
-
-if !FOUND_CPPLINT
-cpplint.py:
-	wget -O cpplint.py $(CPP_LINT_URL)
-	chmod u+x cpplint.py
+  $(error cpplint not found. Install cpplint via pip and re-run configure.)
```

Could we clarify this a bit as to which one it is, maybe something like:
```suggestion
  $(error cpplint not found. Install forked cpplint (e.g. via pip) and re-run configure.)
```

We also can be more packaging agnostic:
```suggestion
  $(error cpplint not found. Install forked cpplint (e.g. via pip for the latest version) and re-run configure.)
```

As it's also in some nice friendly debs:
https://packages.ubuntu.com/kinetic/cpplint

---

Fixed!